resources/skins.citizen.commandPalette/composables/useActionNavigation.js

```
@@ -0,0 +1,115 @@
+		actionButton.focus();
+		state.isActionButtonFocused.value = true;
+		return true;
+	};
+
+	// Event handlers for action button navigation
+	const keydownHandler = ( event ) => {
+		// Check if the event target is an action button
+		if ( event.target.classList.contains( 'citizen-command-palette-list-item__action' ) ) {
+			if ( event.key === 'ArrowLeft' ) {
+				const prevButton = event.target.previousElementSibling;
+				if ( prevButton && prevButton.classList.contains( 'citizen-command-palette-list-item__action' ) ) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)